src/Config.ts

```diff
@@ -177,8 +177,70 @@ var DefaultConfig = {
         return node.data.Name.substring(0, 8)
     },
     groupSize: 5,
-    groupBy: function (node: Node) {
-        return node.data.Type && node.data.Type !== "host" ? node.data.Type : null
+    groupType: function (child: NodeWrapper) {
```

Comment:
I would prefer to not pass the wrapper as it is used internally, node seems to be enough

Reply:
Why do we have groupType and groupGid could it be the same field

nit: I would prefer groupGID

Reply:
pass Node instead of WrapperNode

changed groupGid to groupGID

groupType is separate from groupGid as it is called within Topology to get the type of the group node.